### PR TITLE
Amended lecture indices

### PR DESCRIPTION
**Pull request by jeremy-sanchez**
Lectures 11, 14-40

### CODE REVIEW

File: lectures/11/index.md [hunk 1]
```diff
@@ -14,18 +14,27 @@ links:
   url: https://stat20.org/lectures/00/lecture-title.html
```
Comment (andrewpbray) [inline lectures/11/index.md:14]: Can you also add a summary so that something comes up on the lecture list?

File: lectures/18/index.md [hunk 2]
```diff
@@ -14,6 +14,28 @@ links:
   url: https://stat20.org/lectures/18/hypothesis-testing-concepts.html
 ---
 
+Numerical and graphical summaries might allow us to detect discrepancies between data that has been split by group; for instance, discrepancies in bill depth between Adelie and Chinstrap penguins. However, can we make a more formal statement about whether such discrepancies are statistically different; that is, does species affect bill depth? Today we will show how the hypothesis test comes in handy for this situation!
```
Comment (andrewpbray) [inline lectures/18/index.md:17]: "does species effect bill depth?" The use of causal language here might confuse some students. Perhaps instead "Can we discern whether the difference found in our data would also exist in the more general population?"

File: lectures/11/index.md (hunk 1 above)
Comment (jeremy-sanchez) [reply lectures/11/index.md:14]: Sure thing!

Comment (andrewpbray) [reply lectures/11/index.md:14]: Could you please make it arabic instead of roman numerals?

Comment (andrewpbray) [reply lectures/11/index.md:14]: thanks!

**Review by andrewpbray — APPROVED** [review]
Thanks for doing this!

**Pull request merged**